recipe/meta.yaml

```
@@ -29,20 +27,19 @@ outputs:
         - cmake
         - make  # [unix]
       host:
-        - boost-cpp
+        - libboost-devel
```

Considering that `prrng` consists solely of the following files:
```
include/prrng.h
lib/cmake/prrng/prrngConfig.cmake
lib/cmake/prrng/prrngConfigVersion.cmake
lib/cmake/prrng/prrngTargets.cmake
lib/pkgconfig/prrng.pc
```
There cannot actually be a run-dependence on the boost libraries, so remove them.
```suggestion
        - libboost-headers
```